.github/workflows/build.yml

```diff
@@ -141,21 +141,35 @@ jobs:
     - name: make service dist
       working-directory: service
       run: make -j2 dist
+    - name: make service libs
+      working-directory: service
+      run: make -j2
+    - name: configure runtime dir
+      run: ./autogen.sh && ./configure --disable-openmp --disable-mpi
+    - name: make runtime dist
+      run: make -j2 dist
     - name: set OSC credentials
       run: |
            echo -e "[general]\n\n[https://api.opensuse.org]\ncredentials_mgr_class=osc.credentials.ObfuscatedConfigFileCredentialsManager" > ~/.oscrc
            echo "user = ${{ github.repository_owner }}" >> ~/.oscrc
            echo "pass = ${{ secrets.OSC_CREDENTIALS }}" >> ~/.oscrc
-           if [[ "${{ github.ref_name }}" == "release-v2.0-candidate" ]] || [[ "${{ github.ref_name }}" == "release-v2.0" ]]; then
+           if [[ "${{ github.ref_name }}" == "release-v2.0" ]] || [[ "${{ github.ref_name }}" == "release-v3.0" ]]; then
                echo "OSC_PACKAGE=home:${{ github.repository_owner }}:${{ github.ref_name }}/geopm-service" >> ${GITHUB_ENV}
            else
                echo "OSC_PACKAGE=home:${{ github.repository_owner }}/geopm-service" >> ${GITHUB_ENV}
```

Comment:
OSC_PACKAGE ends in "geopm-service" for both of the publish steps below now.

Won't they clobber each other?